x-pack/plugins/observability/public/utils/is_alert_details_enabled.ts

```diff
@@ -19,11 +26,21 @@ export const isAlertDetailsEnabledPerApp = (
 
```

Comment:
@simianhacker `config` is not available in the context of the APM app. Can you make sure to test whether this works from the alert flyout in the APM app as well?

Reply:
@dgieselaar I refactored `isAlertDetailsEnablePerApp` to work with the APM page, even when the `config` is missing. The `config` bug is an existing bug which is going to require a more involved fixed. I've [filled an issue](https://github.com/elastic/kibana/issues/153141) to address this by the Actionable Observability team for other solutions.  The "Alert Details" button should now be working properly in the alert flyout in APM for the Transaction Latency alerts.